packages/autorest.typescript/test/rlcIntegration/generated/lroRest/src/isUnexpected.ts

```diff
@@ -191,174 +191,174 @@ import {
 } from "./responses";
 
 const responseMap: Record<string, string[]> = {
-  "PUT /lro/put/200/succeeded": ["200", "204"],
   "GET /lro/put/200/succeeded": ["200", "204"],
-  "PATCH /lro/patch/200/succeeded/ignoreheaders": ["200"],
+  "PUT /lro/put/200/succeeded": ["200", "204"],
```

Comment:
just position change.

Reply:
when are you going to fix this issue https://github.com/Azure/autorest.typescript/issues/2264 ?

Reply:
this is not relevant to the #2264 issue, the position changed because we adjust the isUnexcepected implementation a little bit.